### DISCUSSION TITLE
Ide

### DISCUSSION THREAD

**Discussion by mehanalavimajd** [pashmaklang/pashmak#60] [Ideas]
I think pashmak doesn't have any IDE s or code editor.
Shall we create one?

**Comment by parsampsh** [comment]
So. this is not REQUIRED. also creating an ide Is VERY BIG WORK. writing syntax highlighter and auto completer plugins for other IDEs (like vscode) is enough. maybe i write some plugins for them later.

**Comment by amirghazi87** [comment]
Yes , we can not create one , but we can create plugins for code editors or IDEs

**Comment by mehanalavimajd** [comment]
Maybe we can write a pashmak snippets at future
